Livecheckables/mupen64plus.rb

```diff
@@ -0,0 +1,4 @@
+class Mupen64plus
+  livecheck :url   => "https://github.com/mupen64plus/mupen64plus-core/releases",
+            :regex => %r{Latest.*?href="/.*mupen64plus-bundle-src-(\d+(?:\.\d+)+)}m
```

Comment:
```suggestion
            :regex => %r{href=.+/tag/v?(\d+(?:\.\d+)+)}
```

Generally the way we check for versions on a GitHub release page (when necessary) is to check the URLs that contain the tag, so this regex can be simplified/generalized as above.

Just as a side note, usually I would add something like `[/"'` to the end to ensure we're only matching stable versions but we don't have to worry about that here since this is the "latest" page and would implicitly be stable.